organization/ircam_oauth2_provider/views.py

```diff
@@ -0,0 +1,114 @@
+import requests
+from allauth.socialaccount.models import (SocialAccount,SocialLogin)
+from allauth.socialaccount.providers.oauth2.views import (OAuth2Adapter, OAuth2LoginView, OAuth2CallbackView)
+from .provider import IrcamAuthProvider
+from django.conf import settings
+from django.contrib.auth.models import User
+from organization.network.models import Person
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import IntegrityError
+
+import logging
+
+class IrcamAuthAdapter(OAuth2Adapter):
+
+    if not hasattr(settings, 'OAUTH_SERVER_BASEURL'):
+        raise Exception("Couldn't find OAUTH_SERVER_BASEURL in the settings")
+    
+    logger = logging.getLogger('ircamauth')
+    logger.addHandler(logging.StreamHandler())
```

Comment:
The handler should be defined in Django settings.
If it is defined here, there's no way to remove this handler